File: C/Atividade19.c
#include <stdio.h>
#include <conio.h>
#include <stdlib.h>
#include<stdio.h>
#include<time.h> 
#include<string.h>
#include<locale.h>  

int main()
{
	float acel, Vini, tempo, max, conv;
	
	printf ("Aceleracao(m/s2):");
    scanf ("%f", &acel);
	
    printf ("\n\nVelocidade Inicial(m/s):");
    scanf ("%f", &Vini);
	    
    printf ("\n\nTempo(s):");
    scanf ("%f", &tempo);
    
    max=(Vini+(acel*tempo));
    conv= max/3.6; 
	printf ("\n\nO carro esta a uma velocidade de %.2f Km/h", conv);
	
	if (conv<120)
	{
		if (conv<40)
		{
			printf ("\n\nVeiculo muito lento");
		}
		else
		{
			if (conv>80)
			{
				printf ("\n\nVeiculo rapido");	
			}
			else
			{
				if (conv>60)
				{
						printf ("\n\nVelocidade de Cruzeiro");
				}
				else 
				{
						printf ("\n\nVelocidade Permitida");
				}
			}
		}
	}
	else
	{
		printf ("\n\nVeiculo muito rapido");
	}
	
    return 0;
}

File: Python/Atividade19.py
acel = int(input("Aceleração (m/s^2): "))
inicial = int(input("Velocidade Inicial (m/s): "))
tempo = int(input("Tempo de trajeto: "))

MAX= inicial + (acel*tempo)

conv= MAX/3.6

if (conv<120):
    if (conv<40):
        print ("Veículo muito lento!")

    else:
        if (conv>80):
            print ("Veículo rápido!")

        else:
            if (conv>60):
                print ("Velocidade permitida!")

            else:
                print ("Velocidade de cruzeiro!")


else:
    print ("Veículo muito rápido!")